test/monitoring/specs/adobeio_only.json

```diff
@@ -1,9 +1,10 @@
 {
+  "cwd": "franklin-project",
   "parameters": {},
   "output": [
     "setup",
     "--url",
     "https://adobeioruntime.net/api/v1/web/helix/helix-services/monitoring-test@v1/_status_check/healthcheck.json",    "--name",
-    "@adobe/helix-monitoring-test"
+    "@adobe/franklin-monitoring-test"
```

Comment:
`cwd` is the franklin-project, so we expect the project name to have a franklin in it.